docs/calendar.md

```diff
@@ -41,6 +41,8 @@ For specialized applications where you need to do calculations in a calendar sys
 To do this, create a class inheriting from `Temporal.Calendar`, call `super()` in the constructor with a calendar identifier, and implement all the methods.
 Any subclass of `Temporal.Calendar` will be accepted in Temporal APIs where a built-in `Temporal.Calendar` would work.
 
+The identifier of a custom calendar must not be any of the built-in calendar identifiers, and must consist of one or more components of between 2 and 8 ASCII alphanumeric characters each, separated by dashes, as described in [BCP 47](https://tools.ietf.org/html/bcp47#section-2.1).
```

Comment:
Unicode extension value subtags are 3-8, not 2-8

https://unicode.org/reports/tr35/tr35.html#Unicode_locale_identifier

Reply:
Ah, so https://tools.ietf.org/html/bcp47#section-2.1 doesn't contain the full information? Will change this.

Reply:
Yeah; in ECMA-402 we use the UTS 35 version of locale identifiers, which is a subset of IETF.